### ISSUE TITLE
Cannot read property 'protocol' of undefined

### ISSUE THREAD

**Issue by punknroll** [Textalk/angular-schema-form#401]
Hi, I use the development branch because I need a working validation with the checkboxes. After bower update this error occurs. There is an output in the console right in front of the error pointing to line 274 in schema-form.js:

TypeError: Cannot read property 'protocol' of undefined
    at urlIsSameOrigin (angular.js:16442)
    at sendReq (angular.js:9660)
    at $get.serverRequest (angular.js:9383)
    at processQueue (angular.js:13248)
    at angular.js:13264
    at Scope.$get.Scope.$eval (angular.js:14466)
    at Scope.$get.Scope.$digest (angular.js:14282)
    at Scope.$get.Scope.$apply (angular.js:14571)
    at done (angular.js:9698)
    at completeRequest (angular.js:9888)

**Comment by davidlgj** [comment]
Thats strange. It seems to be some kind of origin check on urls? What's on row 274 for you? That is just a `console.log(slots)` for me so it can't be the same.

**Comment by punknroll** [comment]
Hi, the files are the same. I found out that the error is triggered by the addMapping call:

 schemaFormDecoratorsProvider.addMapping('bootstrapDecorator', 'default',  'mydefault.html');

I try to load my custom html. Is this deprecated?

**Comment by punknroll** [comment]
Ok sorry. It is described here:

https://github.com/Textalk/angular-schema-form/issues/386

I hope you will release a stable tag soon where my checkboxes validation is also fixed so I don't have to use the development branch.

**Comment by davidlgj** [comment]
The proposed changes in #386 isn't done yet, but I'm in the middle of redoing how the forms are built to speed it up and to be able to support angular material better (there is work being done on a new decorator). So things have changed a bit. But should be backwards compatible, i.e. add-ons needn't change

But I found a typo that is probably the culprit. I got to go to bed now so I just pushed the fix to development branch without validating if it actually fixes it :) But I think it does. Anyways I'll test it on Friday otherwise.
